src/Microsoft.Performance.SDK/Extensibility/DataCookerPath.cs

```diff
@@ -92,7 +157,11 @@ public DataCookerPath(DataCookerPath other)
         /// <summary>
         /// Gets the combination of the source parser Id and the data cooker Id.
         /// </summary>
-        public string CookerPath { get; }
+        [Obsolete("This will be removed by SDK v1.0.0 release candidate 1")]
+        public string CookerPath
+        {
+            get;
```

Comment:
We can go ahead and put this string as a private field and have the now obsolete property and toString return that.

Reply:
[https://github.com/microsoft/microsoft-performance-toolkit-sdk/blob/feature/cookerPaths-delete/src/Microsoft.Performance.SDK/Extensibility/DataCookerPath.cs](https://github.com/microsoft/microsoft-performance-toolkit-sdk/blob/feature/cookerPaths-delete/src/Microsoft.Performance.SDK/Extensibility/DataCookerPath.cs)

Reply:
yep, Jayson's branch does that. Can I resolve this comment?

Reply:
Or do you want me to bring that change over so we can just delete the property

Reply:
I see the other branch, that's fine.